label.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './item'
+require 'date'
+
+class Label < Item
+  attr_accessor :title, :color, :items
```

Comment:
**[Optional]** Also looking at the [UML class diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png), The `@items` instance variable of the `Label` class is signified with a minus (`-`) sign meaning it should have read-only attribute.

This could mean it should be under the `attr_reader` section.
```
attr_accessor :title, :color
attr_reader :id, :items
```